README.md

```diff
@@ -1,17 +1,12 @@
-Go CSV
-=====
-
-The GoCSV package aims to provide easy serialization and deserialization functions to use CSV in Golang
-
-API and techniques inspired from https://godoc.org/gopkg.in/mgo.v2
+Commando, a CSV library for Go
+==============================
 
-[![GoDoc](https://godoc.org/github.com/gocarina/gocsv?status.png)](https://godoc.org/github.com/gocarina/gocsv)
-[![Build Status](https://travis-ci.org/gocarina/gocsv.svg?branch=master)](https://travis-ci.org/gocarina/gocsv)
+The Commando package aims to provide easy marshalling and unmarshalling of CSV in Go.  It’s a fork of [gocarina/gocsv](https://github.com/gocarina/gocsv), with a simplified API.
 
 Installation
 =====
 
-```go get -u github.com/gocarina/gocsv```
+```go get -u github.com/ieure/commando```
```

Comment:
post-conversation reminder: use `evenco/commando`

Reply:
I fixed it out from underneath you.